modules/util/config/TrainConfig.py

```diff
@@ -863,6 +865,8 @@ def default_values() -> 'TrainConfig':
         data.append(("backup_before_save", True, bool, False))
         data.append(("save_after", 0, int, False))
         data.append(("save_after_unit", TimeUnit.NEVER, TimeUnit, False))
+        data.append(("skip_save", 0, int, False))
+        data.append(("skip_save_unit", TimeUnit.ALWAYS, TimeUnit, False))
```

Comment:
Everything looks correct. I'm just curious about this. Why do we use `TimeUnit.ALWAYS` here instead of `TimeUnit.NEVER` like we do on the `save_after` line above?

Reply:
a default of NEVER would break existing settings. Both conditions have to be true for OT to save.

the default of save_after is NEVER but you change it for example to STEP if you want saving.
if we now add a new condition, the default should be True. ALWAYS is always true until you change it.

Reply:
Ah, that makes sense!